apps/prairielearn/src/pages/instructorGradebook/instructorGradebook.sql

```diff
@@ -158,10 +158,7 @@ FROM
     e.user_id = u.user_id
     AND e.course_instance_id = $course_instance_id
   )
-  LEFT JOIN user_scores AS s ON (u.user_id = s.user_id)
```

Comment:
We don't support multi-column sort right now. Additionally, the default view only shows joined students, so sorting by role isn't that needed. When you change views, the role column becomes visible.